atomic_reactor/plugins/pre_add_labels_in_df.py

```diff
@@ -160,25 +158,23 @@ def generate_auto_labels(self, base_labels, df_labels, plugin_labels):
         for lbl in self.auto_labels:
             if lbl not in generated:
                 self.log.warning("requested automatic label %r is not available", lbl)
-
             else:
-                self.labels[lbl] = generated[lbl]
                 self.log.info("automatic label %r is generated to %r", lbl, generated[lbl])
 
-    def add_aliases(self, base_labels, df_labels, plugin_labels):
-        all_labels = base_labels.copy()
-        all_labels.update(df_labels)
-        all_labels.update(plugin_labels)
-        new_labels = df_labels.copy()
-        new_labels.update(plugin_labels)
+        return {lbl: generated[lbl] for lbl in self.auto_labels if lbl in generated}
```

Comment:
can you just use that for loop above to construct return dict?

Reply:
Updated (also rebased on master)